multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BindServiceToApplicationStep.java

```diff
@@ -44,6 +44,21 @@ protected StepPhase executeAsyncStep(ProcessContext context) throws Exception {
         return StepPhase.POLL;
     }
 
+    private String getBindingName(ProcessContext context, String service, CloudApplicationExtended app) {
+        Map<String, String> bindingNames = context.getVariable(Variables.BINDING_NAMES);
+        String bingingName = null;
+        if (bindingNames.containsKey(service)) {
+            bingingName = bindingNames.get(service);
```

Comment:
again, if you have the binding name + binding params, then you can just get it as 
String bindingName = context.getVariable(Variables.BINDING_NAME);

The binding process is a different diagram (flowable process) which have separate context and won't interfere with the other bindings